docker-compose.yml

```diff
@@ -24,6 +22,8 @@ services:
       - 5432:5432
     environment:
       - POSTGRES_PASSWORD=postgres
+    volumes:
+      - pgdata:/var/lib/postgresql/data
```

Comment:
This might wipe out anyone's local state, before you merge this you should make everyone very aware they should do a backup before pulling down these changes.

Reply:
I've removed it, probably best. We can revisit in a separate PR if any else is hitting the issue.

Reply:
Echoing Nathan: In principle, I do think this is worth having, since it would allow you to preserve data between `down`s but still give you the flexibility to destroy the volume via `down -v` if you want to.